tests/components/test_button.py

```diff
@@ -0,0 +1,81 @@
+"""Tests for the Button Classes."""
+from time import sleep, time
+from typing import List, Type
+
+from j5.backends import Backend
+from j5.boards import Board
+from j5.components.button import Button, ButtonInterface
+
+
+class MockButtonDriver(ButtonInterface):
+    """A testing driver for the button component."""
+
+    def get_button_state(self, board: Board, identifier: int) -> bool:
+        """Get the state of the button."""
+        return False  # Never pushed.
+
+    def wait_until_button_pressed(self, board: Board, identifier: int) -> bool:
+        """Wait until the button was pressed."""
+        sleep(0.2)  # The mock driver always presses the button after 0.2s
+
+
+class MockButtonBoard(Board):
+    """A testing board for the button."""
+
+    @property
+    def name(self) -> str:
+        """The name of this board."""
+        return "Testing Button Board"
+
+    @property
+    def serial(self) -> str:
+        """The serial number of this board."""
+        return "SERIAL"
+
+    def make_safe(self):
+        """Make this board safe."""
+        pass
+
+    @staticmethod
+    def supported_components() -> List[Type['Component']]:
```

Comment:
Why is `Component` in quotes here?

Another occurrence on line 45.

Reply:
No reason.